.monorepolint.config.mjs

```diff
@@ -50,6 +46,56 @@ const JS_TAPE_PACKAGES = TAPE_PACKAGES.filter(
 
 export default {
   rules: [
+    fileContents({
```

Comment:
https://monorepolint.com/docs/rules/standard-tsconfig/ maybe we should use `standardTsconfig` now instead of fileContents. My biggest issue with the current monorepolint is that we've got hand-managed lists of files. Although most of our packages only use a single file, its definitely unexpected if I add another file and the import doesn't work until it gets added in here (and further extends the proliferation of special cases in this file.

Reply:
I wonder if we can get away with something like this and be a little more forgiving of adding extra files.

```json
{
  "include": ["index.js", "index.ts", "lib/*", "*.d.ts"],
  "exclude": ["index.d.ts"]
}
```

Reply:
Will check this idea out. Thanks 👍